jduong09/thirteen
# Issue 5 Pt. 1

Partially handles #5 

## Testing
- Pull down the branch and run `npm install`
- Run `npm run dev` to start the server and navigate to http://localhost:3000
- Click the "Shuffle" button
- 4 individual hands of 13 cards should be randomly shuffled and dealt.
  - This is no longer visible, but you can log it out to see. This should setup passing player 0's hand into the `<Hand>` component for regular gameplay
